### DISCUSSION TITLE
Compatibilidade

### DISCUSSION THREAD

**Discussion by GustavoHoreste** [balta-io/forum#101] [Tira Dúvidas] [answered]
Estou fazendo o curso fundamentos do c# e gostaria de saber se existe algum problema em usar o visual studio 2022 ao inves de baixar .net e utilizar o vs code?

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @GustavoHoreste, como vão as coisas por ai?
Problema nenhum, se você já usa o VS, pode continuar com ele.

Eu uso e recomendo o VS Code no começo por ele ajudar menos... fazendo mais coisas manuais, você fixa mais!
